### PR TITLE
netdata: remove unused imports, fix suggestions

### PR DESCRIPTION
**Pull request by RaghavSood**
###### Motivation for this change

Follow on to #113738, just fixing suggestions highlighted during the review - does not impact the build

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 113764` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/tools/system/netdata/default.nix [hunk 1]
```diff
@@ -36,6 +36,8 @@ in stdenv.mkDerivation rec {
     ++ optionals withSsl [ openssl.dev ];
```
Comment (AndersonTorres) [inline pkgs/tools/system/netdata/default.nix:36]: We need better names for the options!

**Comment by AndersonTorres** [comment]
LGTM. Waiting Borg.

File: pkgs/tools/system/netdata/default.nix (hunk 1 above)
Comment (RaghavSood) [reply pkgs/tools/system/netdata/default.nix:36]: Honestly can't think of a good reason why you'd ever build this without SSL, but I'm going to leave that for a future PR

Comment (SuperSandro2000) [reply pkgs/tools/system/netdata/default.nix:36]: Also the .dev can be removed.

**Pull request merged**